packages/agent/bin/agent.rs

```diff
@@ -53,32 +53,29 @@ struct CliArgs {
 async fn main() {
     let args: CliArgs = CliArgs::parse();
```

Comment:
I think it would be better to put a single conditional here

```rust
#[cfg(not(target_family = "unix"))]
if args.use_linux_path_defaults {
   println!("--use-linux-path-defaults is not supported on Windows");
   std::process::exit(1);
}
```

rather than twice at the different usage points